src/components/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { searchBarContainer, 
+    title, 
+    searchTitle, 
+    underInput, 
+    leftSide, 
+    leftSideItem, 
+    input} from './SearchBar.module.css';
```

Comment:
Unnecessary destructuration. Better use code snippet below for similar cases.
`import styles from './SearchBar.module.css';`
USE:
`<div className={ styles.searchBarContainer }>`